### PR TITLE
ISSD-2247 [PRM] Oppty and Renewal listings are not showing correct data to CAMs

### PR DESCRIPTION
**Pull request by thaynam**

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-fields/19-opportunity-partner-role-sf.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
 		{
 			"DBType": "Boolean",
 			"businessType": "Boolean",
-			"externalReferenceCode": "Active__c",
+			"externalReferenceCode": "Acitve_Partner__c",
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-fields/19-opportunity-partner-role-sf.json:6]:
```suggestion
			"externalReferenceCode": "Active_Partner__c",
```

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/utils/constants/filters.ts [hunk 2]
```diff
@@ -8,11 +8,12 @@ import {getCamelCase} from '../getCamelCase';
 const todayDate = new Date();
 const currentYear = todayDate.getFullYear();
 
-const previousFiscalYearStart = `${currentYear - 1}-01-01`;
-const currentFiscalYearStart = `${currentYear}-01-01`;
 const currentFiscalYearEnd = `${currentYear}-12-31`;
+const currentFiscalYearStart = `${currentYear}-01-01`;
+const previousFiscalYearStart = `${currentYear - 1}-01-01`;
```
Comment (jairmedeiros) [inline workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/utils/constants/filters.ts:13]:
```suggestion
const previousFiscalYearStart = `${currentYear-1}-01-01`;
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/components/Dropdown/Dropdown.tsx [hunk 3]
```diff
@@ -27,8 +27,8 @@ const DropDown = ({className, closeOnClick, icon, label, options}: Props) => (
 		className={className}
 		closeOnClick={closeOnClick}
 		trigger={
-			<ClayButton aria-label="Action Button" displayType="unstyled">
-				{label && (
+			<ClayButton displayType="unstyled">
```
Comment (kylembischof) [inline workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/components/Dropdown/Dropdown.tsx:30]: remove the unneeded tabs or spaces here after `<ClayButton displayType="unstyled">`

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/components/StatusBadge/StatusBadge.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+type StatusClassname = {
+	[key in string]: string;
+};
+
+interface Props {
+	status: string;
+}
+
+const statusClassName: StatusClassname = {
```
Comment (kylembischof) [inline workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/components/StatusBadge/StatusBadge.tsx:18]: can we sort this in alphabetical order?

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/components/Dropdown/Dropdown.tsx (hunk 3 above)
Comment (thaynam) [reply workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/components/Dropdown/Dropdown.tsx:30]: @kylembischof to make things easier I used this PR to push the bug fixes to PROD, this specific change was already in master, but was not suppose to go to PROD yet, it is related to [ISSD-2137](https://liferay.atlassian.net/browse/ISSD-2137)

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/components/StatusBadge/StatusBadge.tsx (hunk 4 above)
Comment (thaynam) [reply workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/components/StatusBadge/StatusBadge.tsx:18]: @eliasantosf pls do this in your PR

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-fields/19-opportunity-partner-role-sf.json (hunk 1 above)
Comment (thaynam) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-fields/19-opportunity-partner-role-sf.json:6]: @jairmedeiros this a API name in Salesforce, I will ask if they can fix it

**Comment by thaynam** [comment]
[resent](https://github.com/liferay-customer-delivery/liferay-portal/pull/24)

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/components/StatusBadge/StatusBadge.tsx (hunk 4 above)
Comment (eliasantosf) [reply workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/components/StatusBadge/StatusBadge.tsx:18]: Hey @kylembischof, the ordination has already taken place in this [PR](https://github.com/is-solutions-delivery/liferay-portal/pull/5518/files). It's even on the master.

**Pull request closed without merging**